src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsoleKeyProcessor.cs

```diff
@@ -184,6 +185,13 @@ protected override int InternalExec(ref Guid pguidCmdGroup, uint nCmdID, uint nC
             {
                 //Debug.Print("Exec: GUID_VSStandardCommandSet97: {0}", (VSConstants.VSStd97CmdID)nCmdID);
```

Comment:
>//Debug.Print("Exec: GUID_VSStandardCommandSet97: {0}", (VSConstants.VSStd97CmdID)nCmdID) [](start = 16, length = 89)

You can probably safely remove this commented out line.

Reply:
Removed.